domain/model/pokemon.go

```diff
@@ -0,0 +1,6 @@
+package model
+
+type Pokemon struct{
+	ID 		int 	`json:"pokedex_number"`
+	Name 	string 	`json:"pokemon_name"`
+}
```

Comment:
Please format the code; give it a look at `gofmt`.

```suggestion
type Pokemon struct{
	ID 	int 	`json:"pokedex_number"`
	Name 	string 	`json:"pokemon_name"`
}
```